web/src/main/java/com/futurewei/alcor/web/entity/AllocationPool.java

```diff
@@ -0,0 +1,37 @@
+package com.futurewei.alcor.web.entity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+@Data
+public class AllocationPool {
```

Comment:
This is an interesting class. How will it be used?

Reply:
this is one property of SubnetWebObject. It's a sub-table